src/types/attachment/interfaces/attachment.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { CommonEntityFields } from '~/types'
+
+export interface Attachment extends CommonEntityFields {
+  author: string
```

Comment:
I guess, you can pick author from user interface

Reply:
done